---
output: github_document
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  fig.path = "man/figures/README-",
  out.width = "100%"
)
```

# ctree <img src='man/figures/logo.png' align="right" height="139" />

<!-- badges: start -->
[![Travis build status](https://travis-ci.org/caravagn/ctree.svg?branch=master)](https://travis-ci.org/caravagn/ctree)
[![Lifecycle: maturing](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://www.tidyverse.org/lifecycle/#maturing)
<!-- badges: end -->

Thee `ctree` package provides a simple implementation of *clones trees* in cancer.

clone trees that can be build of Cancer Cell Franctions (CCFs) clusters computed by tumour subclonal deconvolution. This type of models can be used to study the evolutionary trajectories of a tumour from bulk sequencing data.

Clone trees can be built from Cancer Cell Franctions (CCFs) data reporting fraction of cancer cells harbouring a somatic mutation (usually, a single-nucleotide variant) across a number of tumour regions. This type of models can be used to study the evolutionary trajectories of a tumour from bulk sequencing data, especially in setups where one can generate whole-genome sequencing data.


* `ctree` provides an S3-based implementation of mutation trees, as well as a Monte Carlo sampler that has been discussed in [Caravagna et al; PMID: 30171232](https://www.ncbi.nlm.nih.gov/pubmed/30171232). The package provides also functions to plot and analyze the trees.

* The sibling of a clone tree is a *mutation tree*,  which is obtained from the analysis of binary absence/ presence profiles of somatic variants from bulk targeted panels; mutation trees are implemented in the [mtree package](https://caravagn.github.io/mtree).

*  `mtree` is part of the `evoverse`, a [set of R packages](https://caravagn.github.io/evoverse) to implement Cancer Evolution analyses.

#### Help and support

`ctree` has its own webpage at [GitHub pages](https://caravagn.github.io/ctree/).

-----

### Installation

You can install the released version of `ctree` from
[GitHub](https://github.com/) with:

``` r
# install.packages("devtools")
devtools::install_github("caravagn/ctree")
```

-----

#### Copyright and contacts

Giulio Caravagna, PhD. _Institute of Cancer Research, London, UK_.

* Personal webpage: [https://bit.ly/2kc9E6Y](https://sites.google.com/site/giuliocaravagna/), 
* Email address: [giulio.caravagna@icr.ac.uk](mailto:giulio.caravagna@icr.ac.uk) and [gcaravagn@gmail.com](mailto:gcaravagn@gmail.com)
* Twitter feed: [\@gcaravagna](https://twitter.com/gcaravagna)
* GitHub space: [caravagn](https://github.com/caravagn)
